app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+    # pry
```

Comment:
It's fine for an IC, but on projects and take home challenges for job interviews, you'll want to remove all debugging comments and tools for the final merge.